src/Apps/Order/Routes/NewPayment/index.tsx

```diff
@@ -289,19 +311,21 @@ export const NewPaymentRoute: FC<NewPaymentProps & StripeProps> = props => {
           flexDirection="column"
           style={isLoading ? { pointerEvents: "none" } : {}}
         >
-          {order.mode === "OFFER" && (
-            <>
-              <Flex>
-                <CountdownTimer
-                  action="Submit new payment"
-                  note="Expiration will end negotiations on this offer. Keep in mind the work can be sold to another buyer in the meantime."
-                  countdownStart={order.lastOffer?.createdAt!}
-                  countdownEnd={order.stateExpiresAt!}
-                />
-              </Flex>
-              <Spacer y={[2, 4]} />
-            </>
-          )}
+          {order.mode === "OFFER" &&
+            order.lastOffer?.createdAt &&
+            order.stateExpiresAt && (
```

Comment:
This change was made to address a [no-non-null-assertion](https://github.com/typescript-eslint/typescript-eslint/blob/v2.30.0/packages/eslint-plugin/docs/rules/no-non-null-assertion.md) tslint rule.

I'm not sure what the previous behavior was when either of these was `null`, but I think it makes sense to not display the countdown timer if either of them is `null`.